Mathlib/CategoryTheory/Grothendieck.lean

```diff
@@ -152,6 +162,41 @@ lemma eqToHom_eq {X Y : Grothendieck F} (hF : X = Y) :
     eqToHom hF = { base := eqToHom (by subst hF; rfl), fiber := eqToHom (by subst hF; simp) } := by
   subst hF
   rfl
+
+section Transport
+
+/--
+If `F : C ⥤ Cat` is a functor and `t : c ⟶ d` is a morphism in `C`, then `transport` maps each
+`c`-based element of `Grothendieck F` to a `d`-based element.
+-/
+def transport (x : Grothendieck F) {c : C} (t : x.base ⟶ c) : Grothendieck F :=
```

Comment:
```suggestion
@[simps]
def transport (x : Grothendieck F) {c : C} (t : x.base ⟶ c) : Grothendieck F :=
```

Reply:
I'm unsure whether this would be an improvement. `transport` is not a super-simple definition. If simp unfolds it automatically, expressions might get more complicated quickly.

Reply:
I suggested `simps`, not `simp`.